### DISCUSSION TITLE
Release schedule

### DISCUSSION THREAD

**Discussion by ipotapchuk** [microsoft/openjdk#301] [Q&A] [answered]
Is there any release schedule? I want to know when 17.0.4 will be available.

**Comment by karianna** [comment, marked as answer]
Next Tuesday is our target date.

On Fri, 29 Jul 2022 at 15:58, Illia Potapchuk ***@***.***>
wrote:

> Is there any release schedule? I want to know when 17.0.4 will be
> available.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/microsoft/openjdk/discussions/301>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABME2AJ5ONF5J327QTK3ZLVWPWQ7ANCNFSM55BBFMUA>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Cheers, Martijn (Sent from Gmail Mobile)

**Reply by ipotapchuk** [reply]
> Next Tuesday is our target date.
> 
> On Fri, 29 Jul 2022 at 15:58, Illia Potapchuk ***@***.***>
> wrote:
> 
> > Is there any release schedule? I want to know when 17.0.4 will be
> > available.
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/microsoft/openjdk/discussions/301>, or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/AABME2AJ5ONF5J327QTK3ZLVWPWQ7ANCNFSM55BBFMUA>
> > .
> > You are receiving this because you are subscribed to this thread.Message
> > ID: ***@***.***>
> >
> -- 
> Cheers, Martijn (Sent from Gmail Mobile)
> 

Thanks!
